app/javascript/app/components/input/input-styles.scss

```diff
@@ -0,0 +1,36 @@
+@import '~styles/settings';
+
+.input {
+  width: 100%;
+  height: 45px;
+  border-radius: 4px;
+  background-color: $white;
+  color: $theme-color;
+  border: solid 1px rgba(26, 28, 34, 0.1);
+  padding: 0 40px 0 15px;
+  font-size: $font-size;
+  font-family: $font-family-1;
+  transition: box-shadow 150ms ease-out;
+
+  &::placeholder {
+    color: $theme-color;
+    opacity: 0.5;
+  }
+
+  &:focus {
+    outline: none;
+  }
+
+  &:hover {
+    box-shadow: 0 5px 10px 0 rgba(0, 0, 0, 0.09);
+  }
+
```

Comment:
Would be nice to have a `.disabled` class to change styles in case is needed.  
This is the one that is being used on the dropdown:  
```
&.disabled {
        pointer-events: none;
        cursor: not-allowed;
        background-color: $light-gray;
```